plugins/spotdl.py

```diff
@@ -48,15 +49,19 @@ async def spotify_dl(message: Message):
             return
         await message.edit("Downloading")
         quality = quality.strip()  # Just for Precautions 🤷‍♂
-        cmd = f"spotdl --song {song_n} -o {quality} -f {TEMP_DIR}"
+        cmd = f"cd {TEMP_DIR} && spotdl {song_n}"
```

Comment:
so no use of `quality` ?

Reply:
Spotl v3 is new and doesn't have much args in command line, https://github.com/spotDL/spotify-downloader/blob/master/spotdl/__main__.py#L43

Reply:
ok, thanks :+1: